Exercice_1_-_Le_champ_de_Noel/champ.py

```diff
@@ -34,10 +34,10 @@
 
 def main():
     # initialization
-    # create field with borders
+    # create field with BORDERS
```

Comment:
Attention, ton search and replace a aussi modifié le commentaire : dans le com borders doit rester en minuscules.